packages/shared/src/lib/log.ts

```diff
@@ -67,6 +67,7 @@ export enum Origin {
   WalletPagePackage = 'wallet page package',
   Award = 'award',
   CoresPage = 'cores page',
+  AwardsList = 'awards list',
```

Comment:
cc: @idoshamun added origin for awards list moda

Reply:
perfect